internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -852,3 +905,13 @@ func FindStatusCondition(conditions []hivev1.ClusterDeploymentCondition, conditi
 
 	return nil
 }
+
+// Helper functions to check and remove string from a slice of strings.
+func containsString(slice []string, s string) bool {
```

Comment:
you can use `funk.Contains` no need to define this function.

Reply:
Done